packages/server/src/db/views/staticViews.js

```diff
@@ -25,7 +25,7 @@ const SCREEN_PREFIX = DocumentTypes.SCREEN + SEPARATOR
  * @returns {Promise<void>} The view now exists, please note that the next view of this query will actually build it,
  * so it may be slow.
  */
-exports.createLinkView = async appId => {
+exports.createLinkView = async (appId) => {
```

Comment:
think you might need to run a yarn format or something after pulling in next (after updating the eslint/prettier options).

Reply:
I did indeed, thank you.